src/ApiFunction.ts

```diff
@@ -26,7 +27,7 @@ export class ApiFunction extends Construct {
   constructor(scope: Construct, id: string, props: ApiFunctionProps) {
     super(scope, id);
     // See https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/Lambda-Insights-extension-versionsx86-64.html
-    const insightsArn = 'arn:aws:lambda:eu-west-1:580247275435:layer:LambdaInsightsExtension:21';
+    const insightsArn = `arn:aws:lambda:${props.insightsArnRegion}:580247275435:layer:LambdaInsightsExtension:21`;
```

Comment:
Hier kun je ook gewoon const insightsArn = `arn:aws:lambda:${Stack.of(this).region}:580247275435:layer:LambdaInsightsExtension:21` doen lijkt me?

Reply:
Zeker, zo doen we 't in de andere 2 ook. Dat ding vraagt aan de stack waar de construct in zit welke regio dat ie heeft.